index.html

```diff
@@ -50,6 +50,264 @@ <h1>I'm Matunda. Glad to see you!</h1>
+              <div class="year">
+                <p>2015</p>
+              </div>
+            </div>
+
+          </div>
+          <div class="summary">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          
+            <ul class="tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+           
+          <div class="action">
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
+
+        </div>
+        <!-- Grid card 4 -->
+
+        <div class="grid-card big-card">
+          <div class="snapshot">
+            <img src="./images/grd4.svg">
+          </div>
+          <div class="project-details">
+            <div class="project-name"><h2>Multi-Post<br>Stories         
+            </h2></div>
+          </div>
+            <div class="project-more-details">
+              <div class="client">
+                <p>CANOPY</p>
+                              
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="role">
+                <p>Back End Dev</p> 
+                
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="year">
+                <p>2015</p>
+              </div>
+            </div>
+
+          
+          <div class="summary">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          
+            <ul class="tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+           
+          <div class="action">
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
+
+        </div>
+
+
+      </div>
+    </section>
+    <!--About Me Section-->
+    <section>
+      <div class="about">
+        <div class="top">
+          <h1>About Me</h1>
+          <div class="message">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+          <div class="social-media">
+            <p class="social-media-text">LET'S CONNECT</p>
+            <div class="social-media-icons">
+              <ul>
+              <li><a href="#"><img src="./images/twitter icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Linkedin icon.svg"></a></li>
+              <li><a href="#"><img src="./images/M icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Github icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Chill icon.svg"></a></li>
```

Comment:
[OPTIONAL]
- [ ] Please always remember to add `alt` attribute to an `img` tag. It enables screen readers to read the information about on-page images.